browser/css/toolbar.css

```diff
@@ -321,6 +321,7 @@
 	background-position: right;
 	background-repeat: no-repeat;
 }
+.unospan-options-navigator .unobutton > img,
 .unospan-optionstoolboxdown .unobutton > img {
```

Comment:
maybe we can prepare some generic rule? not target every element 1 by 1?

Reply:
The notebookbar `impress` was created differently from `writer|calc|draw`
Check 

https://github.com/CollaboraOnline/online/blob/cf23fdc2851732a45f94fe6f37f37af967b34738/browser/src/control/Control.Notebookbar.js#L548

https://github.com/CollaboraOnline/online/blob/cf23fdc2851732a45f94fe6f37f37af967b34738/browser/src/control/Control.NotebookbarImpress.js#L66

Reply:
both start with `options`
`[id^="options"]`

Reply:
Used `id="optionstoolboxdown"` since it is the container of the toolbox that encompases the navigator icon across all